lib/rouge/lexers/fortran.rb

```diff
@@ -7,11 +7,11 @@ module Rouge
   module Lexers
     class Fortran < RegexLexer
       title "Fortran"
-      desc "Fortran 95 Programming Language"
+      desc "Fortran 2008 Programming Language"
```

Comment:
Actually, it would make sense to add that this is "free-form" Fortran. What about `desc "Fortran 2008 (Free-Form)"`?